llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1046,23 +1048,73 @@ static void simplifyRecipe(VPRecipeBase &R, VPTypeAnalysis &TypeInfo) {
         assert(TypeInfo.inferScalarType(VPV) == TypeInfo2.inferScalarType(VPV));
     }
 #endif
+    if (VPC)
+      return {VPC};
+    return {};
```

Comment:
```suggestion
    return VPC ? {VPC} : {}
```

Reply:
Unfortunately, the ternary operator in C/C++ is a special case, and this code is invalid: we can't put an initializer list in the RHS of a ternary operator.